CRM/Import/Parser.php

```diff
@@ -273,7 +273,10 @@ protected function getContactFields(string $contactType): array {
    */
   protected function getImportFieldsForEntity(string $entity): array {
     return (array) civicrm_api4($entity, 'getFields', [
-      'where' => [['usage', 'CONTAINS', 'import']],
+      'where' => [
```

Comment:
@jmcclelland I'm not too familiar with this code - could you add a comment to explain the `where`? I feel like if this behaviour changes at some point in the future and custom fields are missing during import, someone will be very confused about why the condition is that way.